.travis.yml

```diff
@@ -4,4 +4,5 @@ node_js:
 before_install:
   - sudo apt-get -qq update
   - sudo apt-get install -y libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential g++
+  - cp config.json.example config.json
```

Comment:
https://docs.travis-ci.com/user/database-setup/#Redis and probably also need to setup mongo